src/Process/Pool/Logger/Message.php

```diff
@@ -17,14 +18,11 @@ class Message implements MessageInterface, \JsonSerializable
 
     protected $time;
     protected $level;
-    protected $process_id;
-    protected $process_path;
-    protected $kojo_job;
-    /** @var SerializableProcessInterface */
-    protected $kojo_process;
```

Comment:
These last two I feel less strongly about removing because they've only been around for a short time, but maybe they should also wait until the next major version

Reply:
Restored with `@deprecated` flag.